### DISCUSSION TITLE
How to PXE boot to Shredos

### DISCUSSION THREAD

**Discussion by horton300** [PartialVolume/shredos.x86_64#137] [General]
Hi I want to create a VM in Hyper-V and PXE boot Shredos from laptops and computers that I want to wipe. please let me know how I can do this

**Comment by PartialVolume** [comment]
@horton300 I know nothing about Hyper-V and have never so far setup a PXE server but others here use PXE for serving up ShredOS. Maybe they are able to help you out.

**Comment by horton300** [comment]
It does not need to be Hyper-V I can install a linux distro too. please advise which distro of linux is better and how to do that. I mean where to copy the files and which files should be copied to the folders. thanks

**Comment by Firminator** [comment]
Try netboot.xyz for all your PXE-booting needs. It's low footprint and doesn't add the complexity of locally setting up and maintaining a PXE boot server, let alone deal with Hyper-V.
Antonym has added ShredOS into the PXE menu (see https://github.com/netbootxyz/netboot.xyz#Utilities)
